docs/plugins/internationalization.md

```diff
@@ -151,13 +151,42 @@ export const myPluginTranslationRef = createTranslationRef({
 
 Step 1: Create translation resources
 
+Option 1 (using the translation messages helper method):
+
+```ts
+// packages/app/src/translations/userSettings.ts
+import {
+  createTranslationMessages,
+  createTranslationResource,
+} from '@backstage/core-plugin-api/alpha';
+import { userSettingsTranslationRef } from '@backstage/plugin-user-settings/alpha';
+
+export const userSettingsTranslations = createTranslationResource({
+  ref: userSettingsTranslationRef,
+  translations: {
+    zh: () =>
+      Promise.resolve({
```

Comment:
I don't think we should recommend this pattern as opposed to using `import`. The `import` makes sure the translations are lazy loaded and this breaks that. In the app it makes sense to sometimes provide messages eagerly, but I think plugin translations should always be lazy.

Reply:
Ok, I will remove this way completely!